packages/jest-message-util/src/index.ts

```diff
@@ -273,6 +274,9 @@ export const getTopFrame = (lines: Array<string>): Frame | null => {
     const parsedFrame = stackUtils.parseLine(line.trim());
 
     if (parsedFrame && parsedFrame.file) {
+      if (parsedFrame.file.startsWith('file://')) {
+        parsedFrame.file = fileURLToPath(parsedFrame.file);
```

Comment:
we should maybe use `slash` as well here, so it's the same on windows and *nix

Reply:
Maybe it should always be executed instead of only when url is detected.

Reply:
Nevermind, fileURLToPath generates a non posix path.
Fixing fileURLToPath output should be enough.